components/Ink/src/private/MDCInkLayer.h

```diff
@@ -74,7 +74,7 @@
 /**
  Starts the ink ripple animation at a specified point.
  */
-- (void)startAnimationAtPoint:(CGPoint)point;
+- (void)startInkAtPoint:(CGPoint)point withAnimation:(BOOL)animated;
```

Comment:
I thought that it was implied that there was an animation. When would we want to do this without animation?

Reply:
Also if we add this don’t forget to update the doc comment above

Reply:
in the new api we need to have ink show up without animation for cards. as an example, if a selected cell gets out of the frame and then is reused afterwards.